library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -80,201 +91,45 @@
  * @tags database, sql, jdbc, partitionable,exactlyOnce
  */
 @Evolving
-public abstract class AbstractJdbcPollInputOperator<T> extends AbstractStoreInputOperator<T, JdbcStore>
-    implements ActivationListener<Context>, IdleTimeHandler, Partitioner<AbstractJdbcPollInputOperator<T>>
+public abstract class AbstractJdbcPollInputOperator<T> extends AbstractStoreInputOperator<T, JdbcStore> implements
+    ActivationListener<OperatorContext>, Partitioner<AbstractJdbcPollInputOperator<T>>
 {
-  /**
-   * poll interval in milliseconds
-   */
-  private static int pollInterval = 10000;
+  private static int DEFAULT_QUEUE_CAPACITY = 4 * 1024 * 1024;
```

Comment:
Perhaps this seems too large for the queue capacity? It means even for a 100 byte record, ~ 400MB of data will be in the queue!